INTITA/Modules/ProfileSettings/SettingsProfileViewController.swift

```diff
@@ -73,46 +87,39 @@ extension SettingsProfileViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         
-        var height: CGFloat = 0
-        
-        if indexPath.row == 0 {
-            height = 188
-        } else {
-            height = 45
-        }
-        
-        return height
+        return 45
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        if indexPath.row == 0 {
-            let headerCell = tableView.dequeueReusableCell(withIdentifier: "HeaderSettingsTableViewCell", for: indexPath) as! HeaderSettingsTableViewCell
-            
-            headerCell.delegate = self
-            return headerCell
+        let cell = tableView.dequeueReusableCell(withIdentifier: InfoSettingProfileTableViewCell.identifier, for: indexPath) as? InfoSettingProfileTableViewCell
+        
+        cell?.aboutSelfLabel.text = arr[indexPath.row] + " : "
+        cell?.isProfileEditing = isProfileEditing
+        
+        if indexPath.row.isMultiple(of: 2) {
+            cell?.backgroundColor = .systemGray6
```

Comment:
Let's have all configuration of the cell inside the cell. You could create a method inside InfoSettingProfileTableViewCell `func configure(with title: String, isEditing: Bool, isEven: true) { ` and set all the content of the cell inside this method. It will look better. Also I suggest to rename `aboutSelfLabel` to something suitable -> `titleLabel`

Reply:
Thanks. I have just done it.